docs/source/devguide/documentation.rst

```diff
@@ -64,87 +31,77 @@ Writing New Documentation
 
-First, the :file:`/sphinx/docs/source` directory contains the reStructuredText source for the Sphinx documentation. In order to compile the Sphinx documentation into HTML, you can run the following from within the :file:`sphinx/docs` directory:
+First, the :file:`/docs/source` directory contains the reStructuredText source for the Sphinx documentation. In order to compile the Sphinx documentation into HTML, you can run the following from within the :file:`docs` directory:
 
 .. code-block:: none
 
-    cd sphinx/docs
+    cd docs
     make html
 
-This command will use the Sphinx tool to generate HTML in the :file:`/sphinx/docs/build` directory. You may use a web browser, such as Mozilla Firefox, to view your newly generated documentation as follows:
+This command will use the Sphinx tool to generate HTML in the :file:`/docs/build` directory. From within the :file:`/docs` directory, you may use a web browser, such as Mozilla Firefox, to view your newly generated documentation as follows:
 
 .. code-block:: none
 
-    cd sphinx/docs/build
-    firefox index.html
-
-Likewise, in order to build a revised version of the :ref:`OpenMOC API <api>` from Doxygen-style comments in the Python and C++ source code, you may use the following console command from within the :file:`sphinx/docs/doxygen` directory:
-
-.. code-block:: none
-
-    cd sphinx/docs/doxygen
-    doxygen Doxyfile
-
-This will build a new version of the API in the :file:`sphinx/doxygen/html` directory. You may use a web browser, such as Mozilla Firefox, to view your newly generated documentation as follows:
-
-.. code-block:: none
+    firefox build/html/index.html
 
-    cd sphinx/doxygen/html
-    firefox index.html
+The reStructuredText source files that should be updated with each pull request are located in the :file:`/docs/source` directory. If developers wish to add new pictures to the documentation, image files should be added to the :file:`/docs/img` directory in either png or svg format. Developers can test out their changes to the documentation by recompiling the Sphinx documentation into HTML and viewing the updated HTML files.
 
-In order to **expose** your newly generated HTML documentation to GitHub such that it will present your website, you must copy the HTML into the main :file:`sphinx` directory:
+Likewise, in order to build a revised version of the :ref:`OpenMOC API <api>` from Doxygen-style comments in the Python and C++ source code, you may use the following console command from within the :file:`docs/doxygen` directory:
 
 .. code-block:: none
 
-    cd sphinx
-    cp -rf docs/build/html/. .
+    cd doxygen
```

Comment:
Does this mean the user will be in the "OpenMOC/docs/doxygen/doxygen" or "OpenMOC/docs/doxygen" directory?